presto-docs/src/main/sphinx/release/release-0.239.rst

```diff
@@ -0,0 +1,51 @@
+=============
+Release 0.239
+=============
+
+General Changes
+_______________
+* Fix  :func:`classification_miss_rate` and :func:`classification_fall_out` functions (:pr:`14740`).
+* Fix NPE in `/v1/thread` end point.
+* Fix an issue where the property ``ignore_stats_calculator_failures`` would not be honored for certain queries that had filters pushed down to the table scan.
+* Fix missing query completion events for queries which fail prior to dispatching.
+* Fix potential infinite loop when the setting ``use_legacy_scheduler`` is set to false.
+* Add aggregation function SET_UNION.
+* Add local disk spilling support for `ORDER BY` syntax.
+* Add local disk spilling support for aggregation functions with `ORDER BY` or `DISTINCT` syntax.
+* Add optimization for cursor projection & filter by extract and compute common subexpressions among all projections & filter first. This optimization can be turned off by session property ``optimize_common_sub_expressions``.
+* Add support for 2 security modes for views. The default `DEFINER` security mode is the same as the previous behavior. Tables referenced in the view are accessed using the permissions of the view owner (the **creator** or.
+* Add support for warning on unfiltered partition keys using `partition-keys-to-warn-on-no-filtering` system property.
+* Add support to optimize min/max only metadata query. This is controlled by existing config ``optimizer.optimize-metadata-queries`` and session property ``optimize_metadata_queries``. Note that enabling this config/session property might change query result if there are metadata that refers to empty data, e.g. empty hive partition.
+* Remove experimental feature to perform grouped execution for eligible table scans and its associated configuration property ``experimental.grouped-execution-for-elligible-table-scans`` and session property ``grouped_execution_for_eligible_table_scans``.
+* Definer** of the view) rather than the user executing the query. In the `INVOKER` security mode, tables referenced in the view are accessed using the permissions of the query user (the **invoker** of the view).
+* Enable ``dynamic-schedule-for-grouped-execution`` by default.  In future releases, we will remove this property, and grouped execution will always use dynamic scheduling.
+* Enable ``grouped-execution-for-aggregation`` and ``experimental.grouped-execution-for-eligible-table-scans`` by default.
```

Comment:
sorry for the confusion.  The commit for `Remove experimental feature to perform grouped execution for eligible table scans ...` was merged after this one, so this note should just say 
```
Enable ``grouped-execution-for-aggregation`` by default.
```